components/blitz/src/omero/gateway/facility/TransferFacility.java

```diff
@@ -49,6 +61,12 @@
 
     /** Reference to the {@link DataManagerFacility} */
     private DataManagerFacility datamanager;
+    
+    /** Reference to the {@link BrowseFacility} */
+    private BrowseFacility browser;
+
+    /** References to currently open pixel stores; key is the pixels id*/
+    private static Map<Long, RawPixelsStorePrx> pixelStores = new ConcurrentHashMap<Long, RawPixelsStorePrx>();
```

Comment:
If the RawPixelsStores are not closed by calling the `closeImage()` method, they will be closed when the Gateway is disconnected, https://github.com/dominikl/openmicroscopy/blob/gateway_followup/components/blitz/src/omero/gateway/Gateway.java#L217 (the Connectors themselves hold references to open services, too).